### PR TITLE
WORK IN PROGRESS: Code cleanup

### PR DESCRIPTION
**Pull request by konstantinjdobler**

### CODE REVIEW

File: packages/HTML.package/CSSDefaultStyleRule.class/instance/printOn..st [hunk 1]
```diff
@@ -1,8 +1,10 @@
 as yet unclassified
```
Comment (konstantinjdobler) [inline packages/HTML.package/CSSDefaultStyleRule.class/instance/printOn..st]: category suggestion: printing

File: packages/HTML.package/DHtmlFormatter.class/instance/parseTextAttributesFor..st [hunk 2]
```diff
@@ -15,5 +15,5 @@ parseTextAttributesFor: aDomNode
 				[:underlined | underlined ifTrue: [ attributes add: TextEmphasis underlined ]];
 		ifStyleNotNil: #struckOut do:
```
Comment (AnjoSs) [inline packages/HTML.package/DHtmlFormatter.class/instance/parseTextAttributesFor..st]: consider to begin the block after the do:

File: packages/HTML.package/CSSFormatter.class/instance/rootValueForEm.st [hunk 3]
```diff
@@ -0,0 +1,3 @@
+as yet unclassified
+rootValueForEm 
+	^20
```
Comment (AnjoSs) [inline packages/HTML.package/CSSFormatter.class/instance/rootValueForEm.st]: Maybe the name could be more descriptive

File: packages/HTML.package/CSSDistanceFormatter.class/instance/put.at.to..st [hunk 4]
```diff
@@ -3,6 +3,7 @@ put: aPropertyValue at: anAttribute to: aContext
...
 	| aPrefixedAttribute |
 	
-	aPrefixedAttribute := self class attributePrefix, '-', anAttribute asSymbol.
-	
-	super put: (self absoluteSizeToPx: aPropertyValue defaultSize: 0) at: aPrefixedAttribute to: aContext
\ No newline at end of file
+	aPrefixedAttribute := self class attributePrefix, '-', anAttribute asSymbol.	
+	self isAbsoluteSize: aPropertyValue ifTrue: [ super put: (self absoluteSizeToPx: aPropertyValue defaultSize: 0) at: aPrefixedAttribute to: aContext].
+	self isRelativeSize: aPropertyValue ifTrue: [ super put: (self relativeSizeToPx: aPropertyValue parentAndDefaultSize:  self rootValueForEm) at: aPrefixedAttribute to: aContext].
+
```
Comment (AnjoSs) [inline packages/HTML.package/CSSDistanceFormatter.class/instance/put.at.to..st]: Those are long blocks. Consider to break the line

File: packages/HTML.package/CSSTextFormatter.class/instance/readTextAlignAttribute.to..st [hunk 5]
```diff
@@ -3,11 +3,11 @@ readTextAlignAttribute: aString to: aContext
...
 		((attribute = 'left') or: [attribute = 'start']) 
-			ifTrue: [ self put: #leftFlush at: #align to: aContext ].
+			ifTrue: [ self put: #leftFlush at: #textAlign to: aContext ].
 		((attribute = 'right') or: [attribute = 'end']) 
-			ifTrue: [ self put: #rightFlush at: #align to: aContext ].
+			ifTrue: [ self put: #rightFlush at: #textAlign to: aContext ].
 		(attribute = 'justify') 
-			ifTrue: [ self put: #justified at: #align to: aContext ]
+			ifTrue: [ self put: #justified at: #textAlign to: aContext ]
 	]
```
Comment (mlichtblau) [inline packages/HTML.package/CSSTextFormatter.class/instance/readTextAlignAttribute.to..st]: C - Style?

File: packages/Scamper.package/HtmlBlockLayout.class/instance/initialize.st [hunk 6]
```diff
@@ -1,7 +1,3 @@
-as yet unclassified
+initialization
```
Comment (mlichtblau) [inline packages/Scamper.package/HtmlBlockLayout.class/instance/initialize.st]: Do we event need this method if we just call super initialize?

**Pull request merged**